app/controllers/posts_controller.rb

```diff
@@ -22,9 +22,24 @@ def index
     @posts = current_user.followed_posts.ordered_posts
   end
 
+  def destroy
+    @post = Post.find(params[:id])
```

Comment:
You should move this to a before action as you are doing it both in `show` and here